samples/boards/mec172xevb_assy6906/qmspi_ldma/prj.conf

```diff
@@ -0,0 +1,9 @@
+# Enable config log
+CONFIG_STDOUT_CONSOLE=y
+
+CONFIG_SPI=y
+CONFIG_SPI_EXTENDED_MODES=y
+CONFIG_SPI_ASYNC=y
+
+CONFIG_PM=n
```

Comment:
@scottwcpg  any consideration to be aware if enabling LPM with this new driver?

Reply:
Which PM?  Soc only, Device, and/or Device Runtime.  Also, in the lowest power state, deep sleep should the pins be disabled?

Reply:
I refer to Soc, CONFIG_PM, why it needs to be disabled in this case?

Reply:
It may be the k_sem_take in spi_context_wait_for_completion().  If the wait time is long enough it may trigger the kernel to enter sleep killing the transaction.  It computes a time in ms based upon the SPI frequency and number of bytes.  The computed time may exceed PM residency.

Reply:
@albertofloyd  See my previous comment.  Test program my be exceeding PM residency triggering sleep entry during a transaction.